docs/howto/openstack/magnum/new-k8s-cluster.md

```diff
@@ -167,63 +221,59 @@ get detailed information about it.
+    | 02e5941a2cf0  |        |          |            |              |               |               |
+    +---------------+--------+----------+------------+--------------+---------------+---------------+
     ```
 
-    For many more details on a specific cluster, note its name and run a
-    command like this:
+    For many more details on a specific cluster, issue a command like this:
 
-    ```bash
-    openstack coe cluster show bangor
-    ```
-
-    ```plain
+    ```console
+    $ openstack coe cluster show $CLUSTER_NAME
     +----------------------+---------------------------------------------------------------------------+
     | Field                | Value                                                                     |
     +----------------------+---------------------------------------------------------------------------+
     | status               | CREATE_COMPLETE                                                           |
     | health_status        | HEALTHY                                                                   |
-    | cluster_template_id  | f9e1a2ea-b1ff-43e7-8d1e-6dd5861b82cf                                      |
-    | node_addresses       | ['192.0.2.105']                                                           |
-    | uuid                 | e0df8c62-c6f6-4c7d-b67e-33e3606e9ab6                                      |
-    | stack_id             | e3725aed-f665-4e8d-9409-85f5ee5e2f4a                                      |
+    | cluster_template_id  | 9ca03308-996e-4eaa-b507-5730dcc19fcc                                      |
+    | node_addresses       | ['198.51.100.5']                                                          |
+    | uuid                 | 7ca7838a-aa33-4259-8784-02e5941a2cf0                                      |
+    | stack_id             | 923c938a-81cd-4a0d-b645-0681ff507bc5                                      |
     | status_reason        | None                                                                      |
-    | created_at           | 2022-11-14T07:32:02+00:00                                                 |
-    | updated_at           | 2022-11-14T07:37:26+00:00                                                 |
-    | coe_version          | v1.18.6                                                                   |
-    | labels               | {'kube_tag': 'v1.18.6', 'heat_container_agent_tag': 'train-stable'}       |
+    | created_at           | 2024-07-05T12:00:41+00:00                                                 |
+    | updated_at           | 2024-07-05T12:08:52+00:00                                                 |
+    | coe_version          | v1.24.16-rancher1                                                         |
+    | labels               | {'container_runtime': 'containerd', 'cinder_csi_enabled': 'True',         |
+    |                      | 'cloud_provider_enabled': 'True', 'docker_volume_size': '20', 'kube_tag': |
```

Comment:
Note: when the useless `docker_volume_size` *label* disappears from the cluster template, we'll need to update this bit.